src/deluge/playback/playback_handler.cpp

```diff
@@ -475,15 +477,15 @@ void PlaybackHandler::endPlayback() {
 
 	bool wasRecordingArrangement = (recording == RecordingMode::ARRANGEMENT);
 
-	bool shouldDoInstantSongSwap =
-	    currentPlaybackMode
-	        ->endPlayback(); // Must happen after currentSong->endInstancesOfActiveClips() is called (ok I can't
-	                         // remember why I wrote that, and now it needs to happen before so that
-	                         // Clip::beingRecordedFrom is still set when playback ends, so notes stop)
+	// Must happen after currentSong->endInstancesOfActiveClips() is called (ok I can't
+	// remember why I wrote that, and now it needs to happen before so that
+	// Clip::beingRecordedFrom is still set when playback ends, so notes stop)
```

Comment:
Here too - basically just in case someone tries to figure out the answer to these questions